src/libs/ajax.js

```diff
@@ -435,18 +436,15 @@ const Groups = signal => ({
 
 
 const Billing = signal => ({
-  listProjects: async () => {
-    const res = await fetchRawls('user/billing', _.merge(authOpts(), { signal }))
-    return res.json()
-  },
+  listProjects: async () => (await fetchRawls('billing/v2', _.merge(authOpts(), { signal }))).json(),
```

Comment:
To match the style of the other functions in this file:
```suggestion
  listProjects: async () => {
    const res = await fetchRawls('billing/v2', _.merge(authOpts(), { signal }))
    return res.json()
  },
```